test/expected/hnsw_index_from_file.out

```diff
@@ -39,7 +39,7 @@ EXPLAIN (COSTS FALSE) SELECT ROUND(l2sq_dist(v, :'v777')::numeric, 2) FROM sift_
 
 SELECT ROUND(l2sq_dist(v, :'v777')::numeric, 2) FROM sift_base1k order by v <-> :'v777' LIMIT 10;
 INFO:  began scanning with 0 keys and 1 orderbys
-INFO:  starting scan with dimensions=128 M=16 efConstruction=128 ef=64
+INFO:  starting scan with dimensions=128 M=16 efConstruction=64 ef=32
```

Comment:
Can you comment on what happened here?

Reply:
I have changed the ef search and ef construction to other values from the default ones we have now, so it would it more visible that the values are being loaded from the file header